.github/workflows/labels.yml

```diff
@@ -0,0 +1,217 @@
+name: Labels
+on:
+  pull_request_target:
+    types: [labeled]
+  issues:
+    types: [labeled]
+
+jobs:
+  handle-labels:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/github-script@v3
```

Comment:
I haven't been locking these particular github actions down to specific SHAs if it's being offered by Github directly. I can change these to specific SHAs if anyone has strong opinions

https://julienrenaux.fr/2019/12/20/github-actions-security-risk/